src/platforms/python/guides/celery/index.mdx

```diff
@@ -115,3 +120,39 @@ You can pass the following keyword arguments to `CeleryIntegration()`:
   See <PlatformLink to="/crons/#celery-beat-auto-discovery">Celery Beat Auto Discovery</PlatformLink> to learn more.
 
   The default is `None`.
+
+## Distributed Traces
+
+Distributes tracing means, that the code running in your Celery tasks will be part of the trace that was started in the code that started your Celery task.
```

Comment:
```suggestion
Distributed tracing extends the trace from the code that's running your Celery task so that it includes the code that initiated the task.
```

Hopefully I have the mental model right here :)